### PR TITLE
feat(starfish): init cache module

### PR DESCRIPTION
**Pull request by DominikB2014**
Init cache module UI behind a flag, don't want to make this PR too large, so will add more stuff in future PRs!

### CODE REVIEW

File: static/app/components/sidebar/index.tsx [hunk 1]
```diff
@@ -276,6 +278,17 @@ function Sidebar() {
                   {...HTTPModuleBadgeProps}
                 />
               </Feature>
+              <Feature features="performance-cache-view" organization={organization}>
+                <SidebarItem
+                  {...sidebarItemProps}
+                  label={
+                    <GuideAnchor target="performance-cache">{t('Cache')}</GuideAnchor>
```
Comment (gggritso) [inline static/app/components/sidebar/index.tsx:285]: Got bitten _twice_ by this already, so going to recommend you set this in `settings.ts`!

File: static/app/views/performance/cache/cacheLandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        <Layout.HeaderContent>
+          <Breadcrumbs
+            crumbs={[
+              {
+                label: t('Performance'),
+                to: normalizeUrl(`/organizations/${organization.slug}/performance/`),
+                preservePageFilters: true,
+              },
+              {
+                label: t('Cache'),
```
Comment (gggritso) [inline static/app/views/performance/cache/cacheLandingPage.tsx:29]: Same note here for using `settings.ts`, and below, as well

File: static/app/components/sidebar/index.tsx [hunk 3]
```diff
@@ -276,6 +278,17 @@ function Sidebar() {
...
+                  }
+                  to={`/organizations/${organization.slug}/performance/cache/`}
+                  id="performance-cache"
+                  icon={<SubitemDot collapsed />}
```
Comment (gggritso) [inline static/app/components/sidebar/index.tsx:289]: Missing `releaseLevelAsBadgeProps` maybe?

**Review by gggritso — APPROVED** [review]

File: static/app/components/sidebar/index.tsx (hunk 1 above)
Comment (DominikB2014) [reply static/app/components/sidebar/index.tsx:285]: 👍

File: static/app/components/sidebar/index.tsx (hunk 3 above)
Comment (DominikB2014) [reply static/app/components/sidebar/index.tsx:292]: Yup, you're right!

**Pull request merged**